pb/api/v1/data.proto

```diff
@@ -11,8 +11,8 @@ import "pb/summary/summary.proto";
 
 service TestGridData {
   // GET /dashboards
-  // Lists dashboard names
-  rpc ListDashboard(ListDashboardRequest) returns (ListDashboardResponse) {}
+  // Lists dashboard names and the dashboard group they belong to
+  rpc ListDashboard(ListDashboardRequest) returns (ListDashboardAndDashboardGroupResponse) {}
```

Comment:
nit: avoid renaming this if possible. It's best if "ListDashboard" returns a "ListDashboardResponse"